mitiq/zne/tests/test_zne.py

```diff
@@ -540,3 +541,22 @@ def execute(circuit: qiskit.QuantumCircuit, shots: int = 8192) -> float:
     # Note: Unmitigated value is also (usually) within 10% of the true value.
     # This is more to test usage than effectiveness.
     assert abs(zne_value - true_value) < 0.1
+
+
+def test_execute_zne_transpiled_qiskit_circuit_with_noise_and_QFT():
```

Comment:
nit: I would rename this test (and the docstring on the line below). 
First of all, starting from the latest iteration, we realised transpiling is not necessary, but only decomposing is. More in general, this test is at a high level and should be unaware of how the circuit is converted under the hood. 
Perhaps just `test_execute_zne_on_qiskit_circuit_with_QFT()` ?

Reply:
Thanks for catching this. I updated the name and doc string as per suggested!